lib/leap_cli/commands/deploy.rb

```diff
@@ -87,14 +87,21 @@ module Commands
     long_desc 'The FILTER can be the name of a node, service, or tag.'
     arg_name 'FILTER'
     command [:history, :h] do |c|
-      c.flag :port, :desc => 'Override the default SSH port.',
-                    :arg_name => 'PORT'
-      c.flag :ip,   :desc => 'Override the default SSH IP address.',
-                    :arg_name => 'IPADDRESS'
+      c.flag   :port, :desc => 'Override the default SSH port.',
+                      :arg_name => 'PORT'
+      c.flag   :ip,   :desc => 'Override the default SSH IP address.',
+                      :arg_name => 'IPADDRESS'
+      c.switch :last, :desc => 'Show last deploy only',
+                      :negatable => false
       c.action do |global,options,args|
+        if options[:last] == true
+          lines = 1
+        else
+          lines = 10
+        end
         nodes = manager.filter!(args)
         ssh_connect(nodes, connect_options(options)) do |ssh|
-          ssh.leap.history
+          ssh.leap.history(lines)
```

Comment:
This could be written in a form that keeps compatibility with the old version of leap_cli:
```ruby
  if ssh.leap.method(:history).arity == 0
    ssh.leap.history
  else
    ssh.leap.history(lines)
  end
```